guides/doc-Managing_Content/master.adoc

```diff
@@ -6,10 +6,14 @@ include::common/header.adoc[]
 = {ContentManagementDocTitle}
 
 ifdef::foreman-deb[]
+== Katello plugin required
+
 The Katello plugin required to manage content in Foreman is not supported on Debian/Ubuntu.
```

Comment:
```suggestion
The Katello plugin required to manage content in Foreman is not supported for Foreman on {DL}.

With Foreman and Katello on EL, you can provide content to hosts running {DL}.
```

Refs https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L71

Out of scope.

Reply:
https://github.com/theforeman/foreman-documentation/pull/3660